README.md

```diff
@@ -218,6 +218,144 @@ OpenStack | N | N | Y | N | N
 oVirt | N | N | Y | N | N
 VMWare | N | N | Y | N | N
 
+# Short lived Credentials with AWS Secure Token Service
```

Comment:
Do we need to mention here that this a preview and not the final version? Something like "beta version"?

Reply:
Mentioning this as a beta feature